src/list_menu.c

```diff
@@ -1261,17 +1261,17 @@ void ListMenuSetUpRedOutlineCursorSpriteOamTable(u16 rowWidth, u16 rowHeight, st
     s32 i, j, id = 0;
 
     subsprites[id] = sSubsprite_RedOutline1;
-    subsprites[id].x = 136;
-    subsprites[id].y = 136;
+    subsprites[id].x = (s8) 136;
+    subsprites[id].y = (s8) 136;
```

Comment:
We format casts as `(s8)136` rather than `(s8) 136`. There's a few other instances in this file, and I think there's a bunch in other files too (usually to do with unsigned vs signed confusion on GF's part).